packages/clients/package.json

```diff
@@ -9,8 +9,10 @@
         "clients"
     ],
     "dependencies": {
+        "@open-condo/cli": "workspace:^",
         "@open-condo/config": "workspace:^",
-        "dotenv": "^16.3.1"
+        "dotenv": "^16.3.1",
```

Comment:
Why? you should use `@open-condo/config`

Reply:
old dependancies was "dotenv": "^16.3.1" 
current PR only adds comma